sklearn/base.py

```diff
@@ -318,13 +318,19 @@ def __getstate__(self):
             state = self.__dict__.copy()
 
         if type(self).__module__.startswith("sklearn."):
-            return dict(state.items(), _sklearn_version=__version__)
+            return dict(
+                state.items(),
+                _sklearn_pickle_version=__version__,
```

Comment:
Looking at this again, I am starting to prefer making this more official by naming the attribute:

```suggestion
                __sklearn_pickle_version__=__version__,
```

and then documenting this in https://scikit-learn.org/stable/model_persistence.html.

Reply:
Using custom dunder attribute names is explicitly discouraged by PEP8. I prefer the original `_sklearn_pickle_version` attribute name.